src/PrestaShopBundle/Form/Admin/Sell/Product/FooterType.php

```diff
@@ -50,21 +51,29 @@ class FooterType extends TranslatorAwareType
      */
     private $router;
 
+    /**
+     * @var ProductPreviewProvider
+     */
+    protected $productPreviewUrlProvider;
+
     /**
      * @param TranslatorInterface $translator
      * @param array $locales
      * @param ProductProvider $productUrlProvider
+     * @param ProductPreviewProvider $productPreviewUrlProvider
      * @param RouterInterface $router
      */
     public function __construct(
         TranslatorInterface $translator,
         array $locales,
         ProductProvider $productUrlProvider,
+        ProductPreviewProvider $productPreviewUrlProvider,
```

Comment:
As the constructor is not the same in 1.7.8.x, could you define the BC Break : https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Form/Admin/Sell/Product/FooterType.php ?

Reply:
done